src/CarbonAware.CLI/src/CarbonAware.CLI.csproj

```diff
@@ -52,11 +52,19 @@
   </ItemGroup>
 
  <Target Name="CopyDataFiles" AfterTargets="Build">
-		<ItemGroup>
-			<DataFiles Include="$(ProjectDir)..\data\data-files\*.*" />
-		</ItemGroup>
+    <ItemGroup>
+      <DataFiles Include="$(ProjectDir)..\..\data\data-files\**\*.*" />
+    </ItemGroup>
 
-		<Copy SourceFiles="@(DataFiles)" DestinationFolder="$(TargetDir)\data-files\" SkipUnchangedFiles="true" />
-	</Target>
+    <Copy SourceFiles="@(DataFiles)" DestinationFolder="$(TargetDir)\data-sources\json\%(RecursiveDir)" SkipUnchangedFiles="true" />
```

Comment:
Do we really want to put every file from the generic `data-files` dir into the `data-sources\json` directory?

Reply:
I'm open for suggestions. Right now this is what the WebApi does too. If we decide to just copy over the one and only file that we consume, that is totally fine. But it would imply to whoever wants to copy another file from the src tree to the bin tree, to change the projects file too (cli and webapi).